charts/gardener/controlplane/values.yaml

```diff
@@ -405,10 +409,6 @@ global:
           bindAddress: 0.0.0.0
           port: 2719
           tls:
-            caBundle: |
-              -----BEGIN CERTIFICATE-----
-              ...
-              -----END CERTIFICATE-----
```

Comment:
Why was this removed?

Reply:
I checked and did not find a usage for the GCM for the CA bundle. 
Am I missing something?